src/jsonstream.cpp

```diff
@@ -5,14 +5,50 @@
 
 
 using namespace simdjson;
-void find_the_best_supported_implementation();
+
+
 
 typedef int (*stage1_functype)(const char *buf, size_t len, ParsedJson &pj, bool streaming);
 typedef int (*stage2_functype)(const char *buf, size_t len, ParsedJson &pj, size_t &next_json);
 
 stage1_functype best_stage1;
 stage2_functype best_stage2;
 
+
+namespace {
```

Comment:
How come `namespace`, btw? Just curious. I don't see it in the code you moved up here.

Reply:
Why the anonymous namespace? Because I want this find_the_best_supported_implementation to be strictly local. It is a hack and I don't want its symbol to ever show up elsewhere. As the todo indicate, it is temporary code that should go away.

Reply:
I have opened an issue https://github.com/lemire/simdjson/issues/471